pkgs/development/embedded/fpga/icestorm/default.nix

```diff
@@ -62,7 +62,7 @@ stdenv.mkDerivation rec {
       FPGAs and providing simple tools for analyzing and
       creating bitstream files.
     '';
-    homepage    = "http://www.clifford.at/icestorm/";
+    homepage    = "https://github.com/YosysHQ/icestorm/";
```

Comment:
I think the homepage has been moved to http://bygone.clairexen.net/icestorm/, but yeah no https...

Reply:
:/